src/data/archive/fall2023/hackschool.yml

```diff
@@ -0,0 +1,57 @@
+- name: Hackschool
+  quarter: Fall 2023
+  mainLink: https://github.com/uclaacm/hackschool-f23
+  tags: ["html", "css", "javascript", "reactjs"]
+  directors:
+    - Abigail Tran
+
+  workshops:
+    - name: "Session 1: HTML/CSS/JavaScript"
+      repo: https://github.com/uclaacm/hackschool-f23/tree/main/Session-1
+      slides: https://docs.google.com/presentation/d/1c8Nu4pWGw-NKXrt5pAI_bARx8VPMfwHraLA_WhCv4HA/edit?usp=sharing
+      tags: ["html", "css", "javascript"]
+      presenters:
+        - Abigail Tran
+
+    - name: "Session 2: Intro to React"
+      repo: https://github.com/uclaacm/hackschool-f23/tree/main/Session-2
+      slides: https://docs.google.com/presentation/d/1Gb42xAZTBypg74G2vSjRguCO3tdFND8psE9W3SrBlIs/edit?usp=sharing
+      youtube: https://youtu.be/TT_FFzMHaZM
+      tags: ["components", "props", "reactjs"]
+      presenters:
+        - James Wu
+
+    - name: "Session 3: Events and State"
+      slides: https://docs.google.com/presentation/d/1k_iWGxCKufU7Qj3IyEKy_2QEMmLpAJe8rMHTSt7qQAw/edit?usp=sharing
+      tags: ["state", "hooks", "event handling", "reactjs"]
+      presenters:
+        - Thomas McGall
+
+    - name: "Session 4: Navigation"
+      repo: https://github.com/uclaacm/hackschool-f23/tree/main/Session-4s
```

Comment:
looks like this is a broken link. Try https://github.com/uclaacm/hackschool-f23/tree/main/Session-4

Reply:
done